bitcoin/src/p2p/message_network.rs

```diff
@@ -32,6 +32,12 @@ pub struct VersionMessage {
     /// The network address of the peer sending the message
     pub sender: Address,
     /// A random nonce used to detect loops in the network
+    ///
+    /// The nonce can be used to detect situations when a node accidentally
+    /// connects to itself. Set it to a random value and, in case of incoming
+    /// connections, compare the value - same values mean self-connection. If
```

Comment:
I originally wanted to make this a paragraph but whatever...

Reply:
No sweat, done - only takes a few seconds.